src/components/node-list/styles/_filter.scss

```diff
@@ -3,29 +3,16 @@
 @import '../../../styles/variables';
 @import './variables';
 
-.kui-theme--dark .pipeline-row__toggle-icon--kind-filter {
+.pipeline-row__toggle-icon--kind-filter {
   &.pipeline-row__toggle-icon--checked {
```

Comment:
The tag row indicators are showing up as black when active on both themes now, instead of blue as in the design.

Reply:
Yeah I think that was due to a last-minute change where I forgot to escape the custom property variables. Fixed!